src/ChatWindow.test.js

```diff
@@ -293,13 +301,21 @@ const testOnUserRequestToAlexaSubmitHandling = (
     const finalState = chatWindowInstance.state;
     const finalMessages = finalState.messages;
 
-    expect(finalMessages.length).toBe(numberOfMessagesAlreadyInState + 2);
-    expect(finalMessages[finalMessages.length - 2]).toEqual(
-      expectedUserMessage
-    );
-    expect(finalMessages[finalMessages.length - 1]).toEqual(
-      expectedAlexaResponse
+    // We should have added userMessage and all of Alexa's responses.
+    const numberOfAlexaResponses = expectedAlexaResponses.size;
```

Comment:
I dont see the use of this constant. You can use expectedAlexaResponses.size wherever applicable. This is much readable than numberOfAlexaResponses which is ambiguous (can be actualAlexaResponses.size).

Reply:
Renamed to numberOfExpectedAlexaResponses to remove ambiguity.